### PR TITLE
accept json=1 and error code

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: lib/handler.js [hunk 1]
```diff
@@ -30,5 +30,10 @@ module.exports = (viewConfig) => {
       }
       throw boom.boomify(err);
     }
+    const viewOutput = h.view(viewConfig.view, result);
+    if (request.query.json === '1') {
```
Comment (jgallen23) [inline lib/handler.js]: Not right, json wouldn't run through view

File: test/test.globals.js [hunk 2]
```diff
@@ -284,3 +287,45 @@ lab.test('varson setting', async() => {
...
+    }
+  ]);
+  server.views({
+    engines: { html: require('handlebars') },
+    path: `${__dirname}/views`
+  });
+  await server.start();
+  // tests
+  const response = await server.inject({ url: '/yaml?json=1' });
+  expect(response.headers['content-type']).to.contain('application/json');
```
Comment (jgallen23) [inline test/test.globals.js]: This test isn't accurate. Needs to test output

File: lib/handler.js (hunk 1 above)
Comment (jgallen23) [reply lib/handler.js]: Also need to check config var.

Comment (jgallen23) [reply lib/handler.js]: We should double check that the config vars are actually still used. Looks like some left overs

File: test/test.globals.js (hunk 2 above)
Comment (orthagonal) [reply test/test.globals.js]: Git doesn't show it here, but I added this: https://github.com/firstandthird/hapi-views/blob/ab8d9e4e1008f7ba60d7c5a408c958ec4e00fe65/test/test.globals.js#L330

File: index.js [hunk 3]
```diff
@@ -3,19 +3,15 @@
 const aug = require('aug');
 const renderHandler = require('./lib/handler.js');
 const defaults = {
-  enableCache: false,
-  serveStale: false,
+  allowJson: true,
```
Comment (jgallen23) [inline index.js]: allowDebugJson

**Pull request merged**